### DISCUSSION TITLE
Taking down the Microsoft CheckedC repos?

### DISCUSSION THREAD

**Discussion by elee-msft** [secure-sw-dev/checkedc#505] [General]
I have a question about the original Microsoft CheckedC repos (hi, @dtarditi!).  Has everything useful been forked to this organization so that we can take down the Microsoft CheckedC repos?  This search link should show everything I'm asking about: https://github.com/search?q=checkedc+owner%3Amicrosoft&type=repositories.  David, you can reach me privately at elee@microsoft.com if you'd like.

**Comment by dtarditi** [comment]
@elee-msft Thanks for reaching out.  I sent you follow-up email.
